packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -140,40 +141,40 @@
     [
       "LedgerEntry",
       {
-        "nth": 1,
+        "nth": 257,
```

Comment:
Why did these fields change?

Reply:
Fixing discrepancies between the `server_definitions` RPC and the `xrpl-codec-gen` script. Tests all pass because these fields aren't used for signing, they only exist for rippled technical reasons.